src/components/pages/Profile.tsx

```diff
@@ -79,19 +88,101 @@ function HoundList({
   return (
     <div className="houndList">
       {hounds
-        ? hounds.map((hound, key) => <HoundCard key={key} hound={hound} width={140} height={140} />)
+        ? hounds.map((hound, index) => (
+            <HoundCard
+              key={index}
+              hound={hound}
+              width={CanvasWidth}
+              height={CanvasHeight}
+              onClick={() => {
+                if (!parent1) {
+                  setParent1(hounds[index]);
+                } else if (!parent2) {
+                  setParent2(hounds[index]);
+                }
+              }}
+            />
+          ))
         : "Loading..."}
     </div>
   );
 }
 
+const CanvasWidth = 140,
+  CanvasHeight = 140;
+
+function BreedSection({
+  parent1,
+  parent2,
+  setParent1,
+  setParent2,
+}: {
+  parent1: HoundInfo | null;
+  parent2: HoundInfo | null;
+  setParent1: Dispatch<SetStateAction<HoundInfo | null>>;
+  setParent2: Dispatch<SetStateAction<HoundInfo | null>>;
+}): JSX.Element {
+  return (
+    <div className="breedSection">
+      <div className="breedSection__hounds">
+        <div className="breedSection__parent1">
+          {parent1 ? (
+            <HoundCard hound={parent1} width={CanvasWidth} height={CanvasHeight} />
+          ) : (
+            <EmptyCanvas width={CanvasWidth} height={CanvasHeight} backgroundColor="#ffd6f7" />
+          )}
+        </div>
+        <i className="fa fa-heart" style={{ color: "#ffd6f7", fontSize: "2rem" }}></i>
+        <div className="breedSection__parent2">
+          {parent2 ? (
+            <HoundCard hound={parent2} width={CanvasWidth} height={CanvasHeight} />
+          ) : (
+            <EmptyCanvas width={CanvasWidth} height={CanvasHeight} backgroundColor="#ffd6f7" />
+          )}
```

Comment:
I think it would be good if we also keep check of `parent1 != parent2`, but I am not sure.